docs/pages/home.tsx

```diff
@@ -0,0 +1,271 @@
+import { css } from '@emotion/react';
+import { spacing, theme, typography, useTheme } from '@expo/styleguide';
+import React, { PropsWithChildren } from 'react';
+import { Container, Row } from 'react-grid-system';
+
+import { Header } from '~/ui/components/Header';
+import { APIGridCell, CommunityGridCell, GridCell, HomeButton } from '~/ui/components/Home';
+import {
+  APICameraIcon,
+  APIListIcon,
+  APIMapsIcon,
+  APINotificationsIcon,
+  CodecademyImage,
+  DevicesImage,
+  QuickStartIcon,
+  SnackImage,
+  WhyImage,
+} from '~/ui/components/Home/resources';
+import { Layout } from '~/ui/components/Layout';
+import { Spacer } from '~/ui/components/Separator';
+import { Terminal } from '~/ui/components/Snippet';
+import { H1, H2, H3, P } from '~/ui/components/Text';
+import {
+  DiscordIcon,
+  DiscourseIcon,
+  GithubIcon,
+  RedditIcon,
+  TwitterIcon,
+} from '~/ui/foundations/icons';
+
+export const CellContainer = ({ children, style }: PropsWithChildren<{ style?: object }>) => (
+  // https://github.com/sealninja/react-grid-system/issues/175
+  <Container
+    fluid
+    style={{ paddingLeft: -15, paddingRight: -15, marginBottom: spacing[8], ...style }}>
+    {children}
+  </Container>
+);
+
+const Description = ({ children }: PropsWithChildren<object>) => (
+  <P style={{ marginTop: spacing[1], marginBottom: spacing[2], color: theme.text.secondary }}>
+    {children}
+  </P>
+);
+
+const Home = () => {
+  const { themeName } = useTheme();
+  const { palette, button, background } = theme;
+  return (
+    <Layout header={<Header />}>
+      <H1 style={{ marginBottom: 8, fontFamily: typography.fontStacks.black, fontWeight: '900' }}>
```

Comment:
One thing I don't love is that we have a wide variety of styling patterns in the docs. In this file, we have inline styles and named variable styles. In other files, we have all caps variable styles using the `css` function.

Is there a pattern you prefer?

This makes me think that (in a future PR) we might consider moving to TailwindCSS. Right now on the website, we used named variables for all styles (no inline styles), but that is for consistency throughout that codebase.

Reply:
I have created the initial version of Home page a while ago, since then I have learnt more about patterns we want to use in the redesign, then then I was iterating over the code, so this file is a bit chaotic now. I have no strong preference about the approach, can adjust myself, we have a chat about those things with @byCedric, which I hope will be able to provide more context for the chosen approach.

IIRC we mostly want to use Emotion (`css` prop) because it more web-oriented (supports pseudo classes, parent and self selectors, etc.), but the new base tags (`H1`, `P`, etc.) doesn't support custom user CSS, using `style` there is an only way to override base styling. Home page is quite specific, and alterations are only valid there, in other hand I could use raw HTML tags (`h1`, `p`, etc.) and then use `css` in there, but I want to reuse as much components as possible.

I will go through the code and replace the `style` with `css` where applicable (for example `GridCell`).

Reply:
Thanks for replacing those. I dont think this PR is a good time to change how we apply CSS across the docs codebase, but thanks for telling me what you think. I want to set a standard here eventually, so that is helpful.